### PR TITLE
Tutorial: Nginx Authentication Preflight Request with Node.js Backend

### PR DESCRIPTION
**Pull request by theonethread**
Created a new tutorial, and also fixed a typo in my previous one.

I have read and understood the Contributor's Certificate of Origin available at the end of 
https://raw.githubusercontent.com/hetzneronline/community-content/master/tutorial-template.md
and I hereby certify that I meet the contribution criteria described in it.
Signed-off-by: Barnabas Bucsy <barnabas.bucsy@gmail.com>

### CODE REVIEW

File: tutorials/nginx-password-protect-content/01.en.md [hunk 1]
```diff
@@ -0,0 +1,768 @@
+---
+SPDX-License-Identifier: MIT
+path: "/tutorials/nginx-password-protect-content"
+slug: "nginx-password-protect-content"
+date: "2021-09-26"
+title: "Password Protecting Web Content in the Hetzner Cloud"
```
Comment (rook59) [inline tutorials/nginx-password-protect-content/01.en.md:6]: To me your title sounds like this tutorials shows the configuration of a simple http basic auth but this is much more complex and detailed. Can you please mention the API validation in the title, slug, path?

File: tutorials/nginx-password-protect-content/01.en.md [hunk 2]
```diff
@@ -0,0 +1,768 @@
...
+```
+[Unit]
+Description=<your-project> nginx auth-proxy
+After=network-online.target
+
+[Service]
+Type=simple
+Restart=on-failure
+RestartSec=5
+User=root
```
[the remark this answers is not in the document]
Comment (theonethread) [reply tutorials/nginx-password-protect-content/01.en.md:707]: Thanks for pointing out, this was a rushed test, and I did not brush it up before committing. Will fix as soon as I have some free time, hopefully in the weekend!

[the remark this answers is not in the document]
Comment (theonethread) [reply tutorials/nginx-password-protect-content/01.en.md]: Made changes using `User=nobody` and added note on how to create designated user if in need with a more sophisticated service.

File: tutorials/nginx-password-protect-content/01.en.md (hunk 1 above)
Comment (theonethread) [reply tutorials/nginx-password-protect-content/01.en.md]: Renamed tutorial folder, and modified title, description and introduction according.

**Pull request merged**